src/pfe/portal/modules/Templates.js

```diff
@@ -23,18 +23,21 @@ const log = new Logger('Templates.js');
 const DEFAULT_REPOSITORY_LIST = [
   {
     url: 'https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json',
-    description: 'Standard Codewind templates',
+    description: 'Extension to develop in containers with an IDE or language of your choice.',
```

Comment:
Could we please make the Codewind description a complete sentence since the Kabanero description is in complete sentences?

I'd rewrite to, "Codewind is an extension to develop in containers with an IDE or language of your choice."

Reply:
Thanks, I've changed it to the suggestion.

Reply:
Approved! Thank you!